packages/driver/cypress/e2e/commands/sessions/sessions.cy.js

```diff
@@ -68,6 +68,22 @@ describe('cy.session', { retries: 0 }, () => {
         .should('eq', 'about:blank')
       })
 
+      it('clears the browser cookie before each run', () => {
```

Comment:
I moved this into a dedicated test, because it was the about:blank traversal that was causing the cookies to be cleared.